File: Lista2/Exercícios_python/Sala/9.py
a = int(input("Digite o valor do número inteiro A: "))
b = int(input("Digite o valor do número inteiro B: "))

if a%4==0 or a%5==0:
    if b%4==0 or b%5==0:
        print(a, "e", b, "são divisíveis por 4 ou por 5")
    else:
        print(a, "é divisível por 4 ou por 5")
else:
    if b%4==0 or b%5==0:
        print(b, "é divisível por 4 ou por 5")
    else:
        print("Nenhum dos números inseridos é divisível por 4 ou por 5")

File: Listas/Lista2/Exercícios_C/Sala/9.c
#include <stdio.h>
int main ()

{
    int a, b;
    printf("Digite o valor do número inteiro A: ");
    scanf("%i", &a);
    printf("Digite o valor do número inteiro B: ");
    scanf("%i", &b);

    if (a%4==0 || a%5==0) {
        if (b%4==0 || b%5==0)
            {printf("%i e %i são divisíveis por 4 ou por 5", a, b); }
        else
            {printf("%i é divisível por 4 ou por 5", a); }
    }
    else {
        if (b%4==0 || b%5==0)
            {printf("%i é divisível por 4 ou por 5", b); }
        else    
            {printf("Nenhum dos números inseridos é divisível por 4 ou por 5"); }
    }
}
